config/noproxy/packit.yml

```diff
@@ -0,0 +1,48 @@
+## The name of the docker network that containers will be attached to.
+## If you want to proxy OrderlyWeb to the host, you will need to
```

Comment:
```suggestion
## If you want to proxy packit to the host, you will need to
```

though you might want to drop this whole comment as it refers to dev_mode which not currently present